tried to add api call

dhamuy opened the pull request:
Hello! I tried to make an API call to a weather app but I couldn't get my API key to work, I think I need a subscription, or maybe make a call to a free version of it. But I had fun learning about how to make an API call! I'm using https://openweathermap.org/api  and I think my problem is calling the [One Call API 3.0] which requires a subscription, so maybe I need to make a call to another one of their API's? If you have some advice on how to navigate this it would be greatly appreciated!

<img width="865" alt="Screenshot 2025-02-27 at 8 38 51 PM" src="https://github.com/user-attachments/assets/ef6cf568-2122-4b0b-8764-05d1dd67ee85" />

dhamuy committed:
tried to add api call

sampoder reviewed (changes requested):
I'm a bit confused - what's the relationship with the cookie clicker?

dhamuy commented:
Ethan (the instructor for last class I think) initially showed us the 15 challenges to complete and one of them was relating to API calls, so I started building that, and then like halfway through class he realized his mistake in showing us the old slides and showed us the new challenges. But by that time I was too deep and I really wanted to make a successful API call and display the data on my website. But no worries at all, I can go back and implement something more related to the cookie game and upload a new PR. I'll get that done before next class.

dhamuy committed:
week4 update

dhamuy commented:
Just updated my code. Feel free to take a look!

dhamuy committed:
finished socketio

dhamuy commented:
Added my week 5 changes too

dhamuy committed:
finished week6

dhamuy commented:
Added week 6 changes, i added unique IDs to people who log in and then i broadcast a message that the given user has joined with their id. I also added timestamps to each message and an updated list format to support indenting. Was super fun!

dhamuy committed:
week7 done

dhamuy committed:
week8

dhamuy committed:
finished week10
